### DISCUSSION TITLE
Can't find useful documentation for cases outside vue/nuxt/nitro apps

### DISCUSSION THREAD

**Discussion by iPy849** [Dave136/vue-email#217] [Q&A] [answered]
I'm coding a vue3 app using a Supabase backend. Supabase use Deno to run edge functions. At first,  I was using vue-email in the vue3 app (I know it's wrong) and it works pefectly, but I need to move email logic to an edge-function.

I can't find any documentation about Deno and vue-email. There is only a [closed issue in this repo](https://github.com/vue-email/vue-email/issues/46), Actually, there are links in NPM website which leads to non existing URLs. 

Where can  I find an updated documentation or an implementation guide?

**Comment by gabalicious** [comment, marked as answer]
It works with node without a bundler. 

render.mjs
```
import { config } from "@vue-email/compiler";
import nodemailer from "nodemailer";
import { emailTailwindConfig } from "./config/email-tailwind.config.mjs";

const templateName = "WeeklyEmail.vue";

const vueEmail = config("./templates", {
  verbose: true, // Shows more detailed logs
  options: {}
});

const transporter = nodemailer.createTransport({
  host: "localhost", // Use Mailpit
  port: 1025,
  secure: false,
  ignoreTLS: true
});

async function renderAndSendEmail() {
  const template = await vueEmail.render(templateName, {
    props: {
      name: "John Doe",
      emailTailwindConfig
    }
  });

  console.log(template.html);

  // Send email using Mailpit
  await transporter.sendMail({
    from: "test@example.com",
    to: "recipient@example.com",
    subject: "Weekly Update",
    html: template.html
  });

  console.log("Email sent! Check Mailpit at http://localhost:8025");
}

renderAndSendEmail().catch(console.error);
```

https://vue-email.vercel.app/getting-started/examples

**Reply by gogokimc** [reply]
Spent 2 days looking into the official documentation and still can't get the module to work. 
5 mins reading your comment and I got it to work. 
Thanks @gabalicious 

The official Vue-email document needs some updating.
